src/packages.jl

```diff
@@ -81,23 +81,35 @@ module Packages
 import ...GAP: Globals, julia_to_gap, GAPROOT
 
 """
-    load(desc)
+    load(desc::String, version::String = "")
 
-TODO: write text, link to GAP's LoadPackage
+Try to load the newest installed version of the GAP package with name `desc`.
+Return `true` if this is successful, and `false` otherwise.
 
-TODO: support version qualifier?
+The function calls [GAP's `LoadPackage` function](GAP_ref(ref:LoadPackage));
+the package banner is not printed.
 """
-function load(desc::String)
-    return Globals.LoadPackage(julia_to_gap(desc), false)
+function load(desc::String, version::String = "")
+    return Globals.LoadPackage(julia_to_gap(desc),
+                               julia_to_gap(version), false) == true
     # TODO: can we provide more information in case of a failure?
     # GAP unfortunately only gives us info messages...
 end
 
 """
-    install(desc)
+    install(desc::String, interactive::Bool = true)
 
-TODO: copy text from <https://gap-packages.github.io/PackageManager/doc/chap2.html> resp.
-link to that
+Download and install the newest released version of the GAP package
+given by `desc` in the `pkg` subdirectory of GAP's build directory
+(variable `GAP.GAPROOT`).
+Return `true` if the installation is successful or if the package
+was already installed, and `false` otherwise.
+
+`desc` can be either the package name or an URL of an archive or repository
+of the package, or an URL of its `PackageInfo.g` file.
```

Comment:
```suggestion
`desc` can be either the name of a package or the URL of an archive or repository
containing a package, or the URL of a `PackageInfo.g` file.
```
We might also want to rename `desc` to `spec`, to align our terminology with that of `Pkg.jl`